drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -293,7 +293,17 @@ Error AudioDriverPulseAudio::init() {
 	pa_ml = pa_mainloop_new();
 	ERR_FAIL_COND_V(pa_ml == nullptr, ERR_CANT_OPEN);
 
-	pa_ctx = pa_context_new(pa_mainloop_get_api(pa_ml), "Godot");
+	String context_name;
+	if (Engine::get_singleton()->is_editor_hint()) {
+		context_name = "Godot Editor";
+	} else {
+		context_name = GLOBAL_GET("application/config/name");
```

Comment:
Does it need any validation? I.e. are there characters allowed in project names which would break PulseAudio? (E.g. `\n`, or advanced unicode stuff).

Reply:
Probably all the precautions here. https://www.python.org/dev/peps/pep-3131/

Reply:
> Probably all the precautions here. https://www.python.org/dev/peps/pep-3131/

It's essentially the same UAX 31 implemented in #53956, with some extra Python specific restrictions.

Reply:
Well I meant we need to check what the X11 API requires. It might be UAX 31, or it might be plain ascii, or something else. That should be in its documentation.

Reply:
> That should be in its documentation.

Or not... They didn't bother to specify if there's any restriction: https://freedesktop.org/software/pulseaudio/doxygen/context_8h.html#a2784c754947a97f02c78b73d7b1c2d5f

Reply:
Just tested with `\n` and `\0`, and there were no issues (other than `\0` ending the string early as expected). Surprisingly, `\n` even renders properly on KDE.
![image](https://user-images.githubusercontent.com/14253836/140626642-6df84480-740f-4f63-9898-cd31832e5369.png)